src/components/ItemForm.js

```diff
@@ -1,17 +1,41 @@
-import React from "react";
+import React, { useState } from "react";
 import { v4 as uuid } from "uuid";
 
-function ItemForm(props) {
+function ItemForm({ onItemFormSubmit }) {
+  const [formData, setFormData] = useState({
+    id: '',
+    name: '',
+    category: ''
+  })
+
+  const handleChange = (e) => {
+    const key = e.target.name
+    setFormData({
+      ...formData,
+      id: uuid(),
```

Comment:
Instead of setting a `uuid` every time the form changes, perhaps setting the ID on submit would make more sense. What do you think?